kv/txn_coord_sender.go

```diff
@@ -296,15 +317,16 @@ func (tc *TxnCoordSender) sendOne(call client.Call) {
 		var txnMeta *txnMetadata
 		if txnMeta, ok = tc.txns[string(header.Txn.ID)]; !ok {
 			txnMeta = &txnMetadata{
-				txn:             *header.Txn,
+				txn:             header.Txn,
```

Comment:
I don't think it's just tests - I could see us using a sync.Pool in the rpc server and reusing proto objects that way.